src/ucp/core/ucp_request.inl

```diff
@@ -343,6 +350,15 @@ void ucp_request_recv_generic_dt_finish(ucp_request_t *req)
     }
 }
 
+static UCS_F_ALWAYS_INLINE void
+ucp_request_send_state_clear(ucp_request_t *req,
+                             uct_completion_callback_t comp_cb)
+{
+    req->send.state.uct_comp.func   = comp_cb;
+    req->send.state.uct_comp.count  = 0;
+    req->send.state.uct_comp.status = UCS_OK;
+}
```

Comment:
need to set `req->id` and `req->send.uct.func` as well because of https://github.com/openucx/ucx/pull/6626

Reply:
what default values should be used?

Reply:
added evaluation of these values